.opencode/tool/ollama-queue.ts

```
@@ -0,0 +1,1116 @@
+  args: {
+    action: tool.schema
+      .enum(['stats', 'clear', 'clear-expired', 'performance-analysis'])
+      .describe('Cache management action'),
+  },
+  async execute({ action }) {
+    switch (action) {
+      case 'stats':
+        const totalSize = Array.from(modelCaches.values()).reduce(
+          (sum, cache) => sum + cache.size,
+          0,
+        );
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)